common/combat_tactics/00_combat_tactics.txt

```diff
@@ -365,34 +467,43 @@ volley_tactic = {
 			leader = {
 				martial = 16
 			}
-			light_troops = { 
-				who = archers
-				value = 0.3
+			OR = {
+				light_troops = {
+					who = archers
+					value = 0.3
+				}
+				light_troops = {
+					who = siege_machines
+					value = 0.3
+				}
 			}
 		}
 	}
 
+	# light_cavalry
 	light_cavalry_offensive = -1.5
-	camel_cavalry_offensive = -1.5
 	horse_archers_offensive = -1.5
+	animals_offensive = -1.5
+	
 	archers_offensive = 2.0
+	siege_machines_offensive = 2.0
 	enemy = {
 		group = harass
 		factor = 1
 	}
 }
 
-swarm_volley_tactic = {
+swarm_volley_tactic = { #Siege machines replace horse_archers as 'mobile' archers
```

Comment:
I think we need add horse_archers back here

Reply:
Horse archers are back!